src/assets/scripts/client/math/flightMath.js

```diff
@@ -203,8 +203,9 @@ export const calculateTurnInitiaionDistance = (aircraft, currentWaypointPosition
     const nominalBankAngleDegrees = 25;
     const speed = kn_ms(aircraft.speed);
     const bankAngle = degreesToRadians(nominalBankAngleDegrees);
+    const nextWaypointModel = aircraft.fms.currentLeg.nextWaypoint;
```

Comment:
That should be easy enough. You can just modify the return of that method and update the tests.